src/components/Header/Header.scss

```diff
@@ -0,0 +1,32 @@
+.App-header {
+  display: flex;
+  flex-direction: row;
+  background-color: #3F51B5;
```

Comment:
Ideally, colors should be defined in a separate file and referenced as variables, e.g. `$header-background-color` :)